include/SFML/Graphics/Text.hpp

```diff
@@ -159,18 +159,61 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     void setStyle(Uint32 style);
 
     ////////////////////////////////////////////////////////////
-    /// \brief Set the global color of the text
+    /// \brief Set the fill color of the text
     ///
-    /// By default, the text's color is opaque white.
+    /// By default, the text's fill color is opaque white.
+    /// Setting the fill color to a transparent color with an outline
+    /// will cause the outline to be displayed in the fill area of the text.
+    ///
+    /// \param color New fill color of the text
     ///
-    /// \param color New color of the text
+    /// \see getFillColor
     ///
-    /// \see getColor
+    /// \deprecated This function is deprecated and may be removed in future releases.
+    /// Use setFillColor instead.
     ///
     ////////////////////////////////////////////////////////////
     void setColor(const Color& color);
 
     ////////////////////////////////////////////////////////////
+    /// \brief Set the fill color of the text
+    ///
+    /// By default, the text's color is opaque white.
+    /// Setting the fill color to a transparent color with an outline
+    /// will cause the outline to be displayed in the fill area of the text.
+    ///
+    /// \param color New fill color of the text
+    ///
+    /// \see getFillColor
+    ///
+    ////////////////////////////////////////////////////////////
+    void setFillColor(const Color& color);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the outline color of the text
+    ///
+    /// By default, the text's color is opaque black.
+    ///
+    /// \param color New outline color of the text
+    ///
+    /// \see getOutlineColor
+    ///
+    ////////////////////////////////////////////////////////////
+    void setOutlineColor(const Color& color);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the thickness of the text's outline
+    ///
+    /// By default, the outline thickness is 0.
+    ///
+    /// \param thickness New outline thickness, in pixels
+    ///
+    /// \see getOutlineThickness
+    ///
+    ////////////////////////////////////////////////////////////
+    void setOutlineThickness(unsigned int outlineThickness);
```

Comment:
Is there a technical limitation that allows only integral thicknesses?
I'm asking because this API is inconsistent with `Shape::setOutlineThickness(float thickness)`.

(minor thing: I would at least call the parameter the same, i.e. `thickness` instead of `outlineThickness`)